examples/with-ethers/src/index.ts

```diff
@@ -62,12 +63,39 @@ async function main() {
 
   const sentTx = await connectedSigner.sendTransaction(transactionRequest);
 
-  console.log(
-    `Transaction sent!\n\thttps://${network}.etherscan.io/tx/${sentTx.hash}`
+  print(
+    `Sent ${ethers.utils.formatEther(sentTx.value)} Ether to ${sentTx.to}:`,
+    `https://${network}.etherscan.io/tx/${sentTx.hash}`
   );
+
+  if (network === "goerli") {
+    // https://goerli.etherscan.io/address/0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6
+    const wethContract = new ethers.Contract(
+      "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
+      ABI,
+      connectedSigner
+    );
+
+    const wethBalance = await wethContract.balanceOf(address);
+
+    print("WETH Balance:", `${ethers.utils.formatEther(wethBalance)} WETH`);
+
+    const depositTx = await wethContract.deposit({
+      value: ethers.utils.parseEther("0.00001"),
+    });
+
+    print(
+      `Deposited ${ethers.utils.formatEther(sentTx.value)} WETH:`,
```

Comment:
tiniest of tiny nits: might make more sense semantically if we say `Wrapped ${ethers.utils.formatEther(sentTx.value)} ETH`